types/office-js-preview/index.d.ts

```diff
@@ -88558,6 +88558,16 @@ declare namespace Word {
          * [Api set: WordApi 1.1]
          */
         getOoxml(): OfficeExtension.ClientResult<string>;
+        /**
+         * Gets the paragraph by its unique id. Throws an `ItemNotFound` error if the collection is empty.
+         *
+         * @remarks
+         * [Api set: WordApi BETA (PREVIEW ONLY)]
+         * @beta
+         *
+         * @param id Required. Unique Paragraph Id.
```

Comment:
I assume this means it should be a GUID. If that's the case, then I think we should say that.